src/truncated/normal.jl

```diff
@@ -12,6 +12,8 @@ TruncatedNormal
 
 ### statistics
 
+Base.eltype(::Type{<:Truncated{Normal{T},Continuous}}) where {T <: Real} = T
```

Comment:
I'm wondering why this is needed and the generic implementation in https://github.com/JuliaStats/Distributions.jl/blob/3f4856c901b79d183eaf9f1ab469500310fd0182/src/truncate.jl#L76 is not sufficient.

Reply:
I agree, that seems not necessary for me.